lib/statsd_listener.rb

```diff
@@ -5,23 +5,24 @@
 class StatsdListener
   include Term::ANSIColor
 
-  def self.run(port)
+  def self.run(port: nil, host: nil)
     puts "Starting MiniStatsd...\n\n"
 
-    new(port).run
+    new(host: host, port: port).run
   end
 
-  def initialize(port:)
+  def initialize(host: nil, port: nil)
     $stdout.sync = true
 
     @socket = UDPSocket.new
     @port = PortSanitizer.sanitize(port)
+    @host = host
 
-    @socket.bind(nil, @port)
+    @socket.bind(@host, @port)
   end
 
   def run
-    puts "Listening on port #{@port}"
+    puts "Listening on #{@host}:#{@port}"
```

Comment:
So, calling the gem without a host param makes the output go like:

```shell 
$ mini_statsd 
Starting MiniStatsd...

Invalid or no port informed. Assigning default port (8125).

Listening on :8125
```

It'd be nicer to have a proper output on this scenario, like:

```shell
Listening on 127.0.0.1:8126
```